app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    instruction, f_in_name, f_out_name = command.split()
+    if instruction != "cp" or f_in_name == f_out_name:
```

Comment:
it is better to raise informative error so that it is clear to the user if something goes wrong